oshdb/src/main/java/org/heigit/ohsome/oshdb/index/XYGrid.java

```diff
@@ -289,17 +331,17 @@ public int compareTo(IdRange o) {
    * Calculates all tiles, that lie within a bounding-box.
    *
    * <p>
-   * TODO but priority 999: Add possibility to snap the BBX to the tile-grid.
-   * TODO: is an exception needed?
+   * TODO but priority 999: Add possibility to snap the BBX to the tile-grid. TODO: is an exception
+   * needed?
```

Comment:
IMHO it is better to keep the `TODO`s as separate lines

```suggestion
   * TODO but priority 999: Add possibility to snap the BBX to the tile-grid.
   * TODO: is an exception needed?
```